src/satellite/process.ts

```diff
@@ -320,9 +327,9 @@ export class SatelliteProcess implements Satellite {
       SELECT * FROM ${oplog}
         WHERE timestamp IS NOT NULL
           AND rowid > ?
-        ORDER BY rowid ASC
+        ORDER BY rowid ASC;
```

Comment:
Curious but do trailing `;`s matter when passing the sql string for a statement? If so, perhaps we can trim the statement and append one if the string doesn't end in a `;` in a utility `normaliseStatementSql` function?

Reply:
; don't matter for single statement. Some might just have slipped.

I will do another round of review once all questions are solved